### ISSUE TITLE
Is this the correct way to perform the preprocessing on the image

### ISSUE THREAD

**Issue by stereomatchingkiss** [KaiyangZhou/deep-person-reid#303]
```
import torchreid

torchreid.models.show_avai_models()

model = torchreid.models.build_model(name='osnet_ain_x1_0', num_classes=1041)
torchreid.utils.load_pretrained_weights(model, "osnet_ain_x1_0_msmt17_256x128_amsgrad_ep50_lr0.0015_coslr_b64_fb10_softmax_labsmth_flip_jitter.pth")

model.eval()

import torch

#assume input_img  is the image loaded by PIL
input_img = torch.ones(1, 3, 256, 128)

norm_mean = [0.485, 0.456, 0.406] # imagenet mean
norm_std = [0.229, 0.224, 0.225] # imagenet std

input_img /= 255.0
for i in range(0, 3):
    input_img [0, i, :, :] = (input_img [0, i, :, :] - norm_mean[i]) / norm_std[i]

```

According to the [config file](https://github.com/KaiyangZhou/deep-person-reid/blob/master/configs/im_osnet_ain_x1_0_softmax_256x128_amsgrad_cosine.yaml), I guess this should be the correct solution to preprocess the image?

After preprocess, feed the data into the model should have the features?

```
model = torchreid.models.build_model(name='osnet_ain_x1_0', num_classes=1041)
torchreid.utils.load_pretrained_weights(model, "osnet_ain_x1_0_msmt17_256x128_amsgrad_ep50_lr0.0015_coslr_b64_fb10_softmax_labsmth_flip_jitter.pth")

model.eval()

features = model(input_img)
```

Am I doing anything wrong?
Thanks

**Comment by KaiyangZhou** [comment]
at a glance it looks correct, but a simpler way is to create an instance of transform which includes ToTensor and Normalize
